frontend/packages/pipelines-plugin/src/components/repository/list-page/RepositoryRow.tsx

```diff
@@ -43,7 +46,14 @@ const RepositoryRow: React.FC<RowFunctionArgs<RepositoryKind>> = ({ obj }) => {
   return (
     <>
       <TableData className={repositoriesTableColumnClasses[0]}>
-        <ResourceLink kind={referenceForModel(RepositoryModel)} name={name} namespace={namespace} />
+        <ResourceIcon kind={referenceForModel(RepositoryModel)} />
+        <Link
+          to={`${resourcePath(referenceForModel(RepositoryModel), name, namespace)}/Runs`}
+          className="co-resource-item__resource-name"
+          data-test-id={name}
```

Comment:
nit: Can we use data-test instead of data-test-id?

Reply:
I added `data-test-id` here as the pre-defined function is looking for `data-test-id`.